Utils/CameraExtensions.cs

```diff
@@ -0,0 +1,34 @@
+﻿using UnityEngine;
+
+namespace DUCK.Utils
+{
+	public static class CameraExtensions
+	{
+		public static Texture2D TakeScreenshot(this Camera camera, int width, int height, 
+			TextureFormat textureFormat = TextureFormat.ARGB32, int cullingMask = 0)
+		{
+			var renderTexture = new RenderTexture(width, height, 24);
+			var screenShot = new Texture2D(width, height, textureFormat, false);
```

Comment:
minor but here you name it `screenShot` as if shot is another word but in the function name it's `TakeScreenshot`. I think we can consider screenshot to be 1 word